db/schema.rb

```diff
@@ -36,6 +36,5 @@
                   self << "SET search_path TO \"$user\", public"
                   self << "INSERT INTO \"schema_migrations\" (\"filename\") VALUES ('20181116054442_create_factions.rb')"
 self << "INSERT INTO \"schema_migrations\" (\"filename\") VALUES ('20181116054443_create_robots.rb')"
-self << "INSERT INTO \"schema_migrations\" (\"filename\") VALUES ('20181116212150_rename_class_columns_in_robots.rb')"
```

Comment:
This line was basically trash, that migration is one I removed, because I realized I could just rollback create_robots and change it to include the correct class names. Shouldn't have been committed like this, oops! (Normally if I've already shared a migration, I won't update it, but since only I had run create_robots, it was ok to update it.)